src/backend/gl/src/window/egl.rs

```diff
@@ -195,20 +219,26 @@ impl hal::Instance<crate::Backend> for Instance {
             Rwh::Xcb(handle) => handle.window as _,
             #[cfg(target_os = "android")]
             Rwh::Android(handle) => handle.a_native_window,
+            #[cfg(not(target_os = "android"))]
+            Rwh::Wayland(handle) => {
+                let window = wayland::egl_window_create(handle.surface, 1024, 768);
+                window.expect("unsupported window").as_ptr() as _
+                // handle.surface as _
+            }
             other => panic!("Unsupported window: {:?}", other),
         };
         let attributes = [
             egl::RENDER_BUFFER as usize,
             egl::SINGLE_BUFFER as usize,
-            // Always enable sRGB
```

Comment:
if this is only available on EGL 1.5, we need to add a check for 1.5, not just remove them